pkg/util/image_test.go

```diff
@@ -23,7 +23,7 @@ func TestGetRepoCreds(t *testing.T) {
 		{
 			server: "",
 			creds: map[string]api.RegistryCredentials{
-				"index.docker.io": api.RegistryCredentials{
+				"index.docker.io": {
```

Comment:
why remove api.RegistryCredentials? will go infer the type from the map signature?

Reply:
yeah, my IDE suggested it as best practices ;)